src/modules/project/reducer.ts

```diff
@@ -0,0 +1,36 @@
+import { loadingReducer, LoadingState } from 'decentraland-dapps/dist/modules/loading/reducer'
+import { ModelById } from 'decentraland-dapps/dist/lib/types'
+import { Project } from 'modules/project/types'
+import { CreateProjectAction, CREATE_PROJECT } from 'modules/project/actions'
+
+export type ProjectState = {
```

Comment:
Shouldn't this go in `project/types.ts`?

Reply:
We only keep `XXXState` types on the reducer for two things:
- It's close to it's use
- Circular dependencies!

Reply:
Ok, I feel I asked about it before